### DISCUSSION TITLE
Transaction lists

### DISCUSSION THREAD

**Discussion by enigmaticloon** [DSJAS/DSJAS#48] [General Q&A] [answered]
This is an amazing resource I cant wait to see more of it. When adding a transition I understand they are read-only after being created. Is there a way to add transactions in bulk rather than one by one through a config or PHP file? This is especially tricky as the system adds transactions by date and time so the local machine's time has to be adjusted. Not impossible to do but tedious.

**Comment by ejv2** [comment]
Hi. Sorry for the delayed response - I have been very busy recently. I will add a bulk transactions feature and a feature for changing the time to my todo list.

For the time being, I am putting this project in support mode, so major releases will only be worked on for bug fixes and critical security problems, as I have some other very large projects I need to finish for things in my life. Sorry about this. I will keep your suggestion in mind, though, of course.

**Comment by ejv2** [comment, marked as answer]
Hello @enigmaticloon. I know it's been a long time, but there are plans to add a remote API to the site which will allow scripts to remotely interact with the database. I will then perhaps create another repository of contrib scripts (probably Perl or Python or something) which will allow us to do this quite easily. The discussion on this is taking place in #33.

I'm going to close this ticket now and open a feature request issue for it. Thanks for the suggestion!

**Reply by ejv2** [reply]
Your ticket is at #76
